README.md

```diff
@@ -110,13 +110,14 @@ Examples of running the unit tests on ECL, SBCL, and Lispworks are below:
 # I like to wrap these in rlwrap in case I need to use the REPL, but it's optional
 
 #ECL
-rlwrap ecl --load ./run-core-tests.lisp
+rlwrap ecl --load ~/quicklisp/setup.lisp --load ./run-core-tests.lisp
 
 #SBCL
-rlwrap sbcl --load ./run-core-tests.lisp
+rlwrap sbcl --load ~/quicklisp/setup.lisp --load ./run-core-tests.lisp
 
 #Lispworks
 # From the editor run:
+(load #P"~/quicklisp/setup.lisp")
```

Comment:
For lispworks this goes in your init here, with the caveat of the limited personal edition, which I added some more notes too in the readme

more .lispworks

;;; The following lines added by ql:add-to-init-file:
#-quicklisp
(let ((quicklisp-init (merge-pathnames "quicklisp/setup.lisp" (user-homedir-pathname))))
  (when (probe-file quicklisp-init)
    (load quicklisp-init)))